app/main/views/unsubscribe_requests.py

```diff
@@ -0,0 +1,24 @@
+from flask import render_template
+
+from app import current_service
+from app.main import main
+from app.main.forms import ProcessUnsubscribeRequestForm
+from app.utils.user import user_has_permissions
+
+
+@main.route("/services/<uuid:service_id>/unsubscribe-requests/summary")
+@user_has_permissions("view_activity")
+def unsubscribe_request_reports_summary(service_id):
+    return render_template("views/unsubscribe-request-reports-summary.html")
+
+
+@main.route("/services/<uuid:service_id>/unsubscribe-requests/reports/<uuid:batch_id>")
+@user_has_permissions("view_activity")
+def unsubscribe_request_report(service_id, batch_id):
```

Comment:
Think this is what we need to make the page work for unbatched requests:
```suggestion
@main.route("/services/<uuid:service_id>/unsubscribe-requests/reports/<uuid:batch_id>")
@main.route("/services/<uuid:service_id>/unsubscribe-requests/reports/latest")
@user_has_permissions("view_activity")
def unsubscribe_request_report(service_id, batch_id=None):
```

Reply:
Yes, that will make it work.